ansible/vagrant/README.md

```diff
@@ -20,7 +20,7 @@ vagrant plugin install vagrant-openstack-provider --plugin-version ">= 0.6.1"
 Vagrant uses Ansible to automate the Kubernetes deployment. Install Ansible (Mac OSX example):
 ```
 sudo easy_install pip
-sudo pip install ansible
+sudo pip install ansible==2.0.0.2
```

Comment:
Is Ansible 2.0.1.0 have issues with our playbooks/Core OS?

Personally I prefer to specify lowest supported version and hope that newer releases will be better and more stable, so there is no need to add upper limit on version.

So I would do `sudo pip install ansible>=2.0.0.2`, but I don't insist.